src/Chapter1.hs

```diff
@@ -640,8 +651,10 @@ Implement a function that returns the first digit of a given number.
 You need to use recursion in this task. Feel free to return to it later, if you
 aren't ready for this boss yet!
 -}
-
-firstDigit n = error "firstDigit: Not implemented!"
+firstDigit :: Int -> Int
+firstDigit n
+  | n < 10 = n
+  | otherwise = firstDigit (div n 10)
```

Comment:
Great recursion function! Well done 👏🏼 
Almost there! This function should take into consideration negative numbers as well.